drivers/sensor/bma280/bma280_trigger.c

```diff
@@ -15,6 +15,19 @@
 #include <logging/log.h>
 LOG_MODULE_DECLARE(BMA280, CONFIG_SENSOR_LOG_LEVEL);
 
+static inline void setup_int1(struct device *dev,
+			      bool enable)
+{
+	struct bma280_data *data = dev->driver_data;
+
+	gpio_pin_interrupt_configure(data->gpio,
+				     DT_INST_0_BOSCH_BMA280_INT1_GPIOS_PIN,
+				     DT_INST_0_BOSCH_BMA280_INT1_GPIOS_FLAGS
```

Comment:
why are you passing in the DT flags here?

Reply:
Probably because whatever sample I coped that boilerplate from did that.  I'll look through the existing conversions for that error when I do the other conversions.  Fixed here.